.circleci/config.yml

```diff
@@ -2,24 +2,17 @@ version: 2.1
 jobs:
   runtest:
     macos:
-      xcode: "11.4.0"
+      xcode: "12.0"
     working_directory: ~/purchases-ios
     shell: /bin/bash --login -o pipefail
     steps:
       - checkout
       
-      # Carthage
-      - restore_cache:
-          keys: 
-            - carthage-cache-{{ checksum "Cartfile.resolved" }}
+      # CocoaPods
       - run:
-          name: Carthage Bootstrap
+          name: Pod Install
```

Comment:
I'm not doing caching here, but I don't think it'd be worth it anyway - it runs pretty fast. 
<img width="1213" alt="Screen Shot 2020-09-18 at 4 49 54 PM" src="https://user-images.githubusercontent.com/3922667/93640004-44bd6c80-f9d0-11ea-832d-bc88357b7903.png">